### ISSUE TITLE
[VITS TTS] ValueError: Attempt to convert a value (None) with an unsupported type (<class 'NoneType'>) to a Tensor

### ISSUE THREAD

**Issue by alessandropettenuzzo96** [PINTO0309/onnx2tf#350]
### Issue Type

Others

### onnx2tf version number

1.11.2

### onnx version number

1.13.1

### onnxruntime version number

1.13.1

### onnxsim (onnx_simplifier) version number

0.4.17

### tensorflow version number

2.12.0

### Download URL for ONNX

https://drive.google.com/file/d/1-L5bjzwt-3_WgiNW7DqfUF4xHtG2BZ70/view?usp=sharing

### Parameter Replacement JSON

```json
{}
```

### Description

Hi @PINTO0309, thank you for your work on onnx2tf!

1. I'm working on a University project and I have to convert this VITS TTS model from pytorch to Tensorflow2 and then to TFLite in order to run it on a mobile device. I was able to convert it from Pytorch to ONNX but I'm stuck converting the resulting model to TF2. I've already tried to solve this on my own for 2 months but with no success. It's very important to me to solve this problem.
2. During the model conversion to TF, using `onnx2tf -i /content/drive/MyDrive/VITS_ONNX/vits.onnx -oh5` the converter stops when trying to convert `/duration_predictor/RandomNormalLike` with error: `ValueError: Attempt to convert a value (None) with an unsupported type (<class 'NoneType'>) to a Tensor`. I don't know what to do in order to solve this issue, I already tried all the solutions that the converter suggests. I also read that `RandomNormalLike` is supported by your converter
3.  I've attached the .ipynb containing all the steps to reproduce the issue and a screenshot of the error:

[ONNX2TF.ipynb.zip](https://github.com/PINTO0309/onnx2tf/files/11454365/ONNX2TF.ipynb.zip)
<img width="1207" alt="Screenshot 2023-05-11 alle 18 25 06" src="https://github.com/PINTO0309/onnx2tf/assets/21243264/2d502d56-c763-42fe-9481-9f4f582594c9">

**Comment by PINTO0309** [comment]
Unfortunately, TensorFlow's `RandomNormalLike` does not seem to accept tensors with only undefined dimensions. This has been a problem in the past when converting speech processing models as you have.

![image](https://github.com/PINTO0309/onnx2tf/assets/33194443/427acb6f-45a7-494c-b7b9-01647e2d3a6c)

See: [[MMVC] Incorrect tensor shape when converting a model with "torch.randn_like" #253](https://github.com/PINTO0309/onnx2tf/issues/253)

The following errors are not errors detected by onnx2tf, but errors detected internally by TensorFlow.

```
Traceback (most recent call last):
  File "/home/b920405/git/onnx2tf/onnx2tf/utils/common_functions.py", line 282, in print_wrapper_func
    result = func(*args, **kwargs)
  File "/home/b920405/git/onnx2tf/onnx2tf/utils/common_functions.py", line 360, in inverted_operation_enable_disable_wrapper_func
    result = func(*args, **kwargs)
  File "/home/b920405/git/onnx2tf/onnx2tf/ops/RandomNormalLike.py", line 70, in make_node
    generator.normal(
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/ops/stateful_random_ops.py", line 667, in normal
    shape = _shape_tensor(shape)
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/ops/stateful_random_ops.py", line 202, in _shape_tensor
    return ops.convert_to_tensor(shape, dtype=dtype, name="shape")
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/profiler/trace.py", line 183, in wrapped
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/framework/ops.py", line 1642, in convert_to_tensor
    ret = conversion_func(value, dtype=dtype, name=name, as_ref=as_ref)
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/framework/constant_op.py", line 344, in _constant_tensor_conversion_function
    return constant(v, dtype=dtype, name=name)
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/framework/constant_op.py", line 268, in constant
    return _constant_impl(value, dtype, shape, name, verify_shape=False,
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/framework/constant_op.py", line 280, in _constant_impl
    return _constant_eager_impl(ctx, value, dtype, shape, verify_shape)
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/framework/constant_op.py", line 305, in _constant_eager_impl
    t = convert_to_eager_tensor(value, ctx, dtype)
  File "/usr/local/lib/python3.8/dist-packages/tensorflow/python/framework/constant_op.py", line 103, in convert_to_eager_tensor
    return ops.EagerTensor(value, ctx.device_name, dtype)
ValueError: Attempt to convert a value (None) with an unsupported type (<class 'NoneType'>) to a Tensor.
```

In conclusion, this is more of a problem with TensorFlow itself than with onnx2tf.

A practical solution is to customize the model so that `RandomNormalLike` can be entered into the input OP as a random value according to a normal distribution. 
![image](https://github.com/PINTO0309/onnx2tf/assets/33194443/0969f664-7be5-40ff-985c-5fb9d1631a47)
